.vscode/settings.json

```diff
@@ -3,6 +3,8 @@
   "css.validate": false,
   "scss.validate": false,
   "editor.formatOnSave": true,
-  "prettier.stylelintIntegration": true,
-  "prettier.eslintIntegration": true
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true,
+    "source.fixAll.stylelint": true
```

Comment:
Remove autofixing of stylelint for now, it's got some problems thanks to [upstream bugs with merged but unreleased fixes](https://github.com/stylelint/stylelint/issues/4449). I would say keep it and add js/ts files to the stylelint ignore but that just causes [other problems](https://github.com/stylelint/vscode-stylelint/issues/40)